cap/modules/schemas/serializers.py

```diff
@@ -40,6 +40,7 @@
     'record_mapping',
     'deposit_options',
     'record_options',
+    'config'
```

Comment:
remove for now till we decide what we will return, some configs/info should be private